.gitignore

```diff
@@ -41,7 +41,6 @@ coverage.txt
 # act
 .secrets
 
-/data*
```

Comment:
[q] Why are you removing this, and creating a folder `data` with a gitignore inside instead?

Reply:
I can remove this, but then users will have to create the folder every time they install it

Reply:
I added in a method for generating the folder/clearing it upon start up of a new session. Feel free to compare to your method and choose the best.